### DISCUSSION TITLE
Deactivate tenancy for a certain resource

### DISCUSSION THREAD

**Discussion by juliangums** [filamentphp/filament#9117] [Help] [answered]
How do I disable multi-tenancy for a certain resource? For example, I have an Organization and Offices. The Office is the tenant model. I have a permission for users to be able to change the organization details/settings, but as that model does not belong to an office, I'd like to have this as a tenant-independent resource. Ideally, the tenant slug shouldn't be there, and it should not ask me to provide a tenant owner relationship. Sure, I can create a separate panel for this, but that's not really an option. I want this to be seamless in one panel. Pages might be an option, but I haven't figured out how to do this yet. Any ideas?

**Comment by leandrocfe** [comment]
What code did you try? Could you share more info please?

**Reply by juliangums** [reply]
It's not really related to any custom code. When setting up tenancy, it seems like all resources are expected to provide a tenant relationship and a tenant scope is applied to it. How can I have a resource in a panel that uses tenancy that is excluded from tenancy scoping, as in it is global and all tenants can access it and it does not have a tenant relationship?

**Comment by jstenquist** [comment, marked as answer]
I'm unsure if this is exactly what you're looking for, but try adding this to OrganizationResource.php

`protected static bool $isScopedToTenant = false;`

**Reply by juliangums** [reply]
@jstenquist it's exactly what I was looking for. At the time of asking the question, this didn't exist yet. Glad it's been added. Thank you!
